tools/release

```diff
@@ -49,9 +50,10 @@ cd(tooldir)
 
 if 'upload' in sys.argv:
     cd(distdir)
-    #print( 'Uploading distribution files to GitHub...')
 
-    for fname in os.listdir('.'):
+    # do not upload OS specific files like .DS_Store
+    to_upload = glob('*.whl')+glob('*.tgz')
```

Comment:
I think it should be `*.tar.gz` rather than `*.tgz`.

Reply:
The rest of the script have things like `sh('scp `ls -1tr *tgz | tail -1` %s' % backup_site)` does that mean it's wrong as well ?

Reply:
The tarballs that get uploaded to PyPI are `.tar.gz`. The script also makes a 'backup' tarball using `git archive` (this is done in `make_tarball.py`). That's what has the `.tgz` extension. It gets uploaded to http://archive.ipython.org/release/

I think the 'backup' files are a remnant from the distant past before IPython was in proper version control, to preserve a copy of the code as of that release. I'd like to make our release process simpler and more standard - we trust the combination of Github and PyPI to preserve releases of all our other projects. But more backups always seem like a good idea, and it's mostly automated (once you have the necessary SSH key...), so I've not pushed for removing it.

Reply:
@Carreau ping?